internal/binder/function/funcs_array.go

```diff
@@ -29,4 +35,99 @@ func registerArrayFunc() {
 			return nil
 		},
 	}
+	builtins["array_position"] = builtinFunc{
+		fType: ast.FuncTypeScalar,
+		exec: func(ctx api.FunctionContext, args []interface{}) (interface{}, bool) {
+			array, ok := args[0].([]interface{})
+			if !ok {
+				return errorArrayArgumentError, false
+			}
+			for i, item := range array {
+				if item == args[1] {
+					return i + 1, true
+				}
+			}
+			return 0, true
+		},
+		val: func(ctx api.FunctionContext, args []ast.Expr) error {
+			return ValidateLen(2, len(args))
+		},
+	}
+	builtins["slice"] = builtinFunc{
```

Comment:
This one is already supported by [x:y]? Please check.

Reply:
It seems `slice` is equal to `subset`, but I think it's ok to implement `slice` here.

Reply:
What's the rationale here? To keep the core small, I would recommend not adding any redundant features.

Reply:
updated.